roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -521,6 +521,9 @@ resources:
               params:
                 cluster_id: {{ stack_name }}
                 k8s_type: {{ etcd_hostname | default('etcd') }}
+{% if openshift_openstack_private_dns_domain != openshift_openstack_public_dns_domain %}
```

Comment:
Also, can we just skip the if altogether? And assume that if private is not set, it will default to public?

That should make the templates simpler and not harm anything.

Reply:
I'll try to rework this, good idea